test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -196,7 +196,9 @@ void static command_line_arg_check(int argc, char *argv[],
                     "[-k (kilobytes/second)] [-b (bytes/second)] \n"\
                     "[-v (validate data stream)] \n");
         }
+#ifndef VERSION_1.0
```

Comment:
You may want this to be a proper number, like 100 (where XXYY is XX is the major number and YY is the minor number), in case you have multiple of these checks down the line.